sky/clouds/os_image/README.md

```diff
@@ -0,0 +1,14 @@
+# Custom OS Image Generation for SkyPilot
```

Comment:
How about we move the contents of this folder into `clouds/service_catalog/images`?

Reply:
Done